src/ui/atoms/text/Text.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames'
+import React from 'react'
+import type { DeepReadonly } from '../../../superTypes'
+import './text.scss'
+
+export type TTextProps = DeepReadonly<{
```

Comment:
Based on the @ccamel [comment](https://github.com/okp4/ui/pull/81#discussion_r843176700) the convention dictate to not prefix types with `T`.